### ISSUE TITLE
Drill press speed guide

### ISSUE THREAD

**Issue by Duehok** [comakingspace/do-something#245]
The drill presses rotating speed can (and should) be adjusted. There is currently no guide quickly available to select the correct speed. At least one should be printed out and available near the drill presses.

- [x] find a guide, ideally from a credible source
- [ ] check the licence or get permission for reproducing the guide
- [ ] get it in the wiki
- [x] find a place for it near the drill presses
- [x] print it out in a durable format, or print out a lot of them
- [ ] describe in the wiki page about the drills that the guide exists

**Comment by Duehok** [comment]
guide example:
https://www.sserc.org.uk/wp-content/uploads/2011/12/drill-speed-chart.pdf

**Comment by LukasFreeze** [comment]
👍 good idea! I suggested this guide in #150 a while™️  ago, it's more visual and focuses on the most common types of drill bits: https://www.ellmitron.de/media/Uploaded/static/doc/drehzahltabelle-bohren_3366.pdf

Printing such a document once or twice should be fine in my opinion, but we should not copy & upload the digital file for copyright reasons (linking the original source in the wiki should also be no problem).

For durable prints: we do have a [laminator](https://wiki.comakingspace.de/Laminator), probably living on the shelf with the stereo :-)

**Comment by produkt-manager** [comment]
Es gibt auch solche Onlinerechner für verschiedene Materialien https://www.flott.de/service/drehzahltabelle-drehzahlrechner/, die mindestens in den Wiki passen würden.

**Comment by LukasFreeze** [comment]
Nice, I like the idea :-)

Machine specific stuff is a bit more difficult to place well (not just some
wall in the area), but then it's probably read/followed more often!

**Comment by kFrolichs** [comment]
Since it is a pretty small footprint we could probably just tape it to the cover of the drill press?

**Comment by kFrolichs** [comment]
I have printed and attached the two guides (one for wood the other for metal) on the cover of the Scheppach drill press.
They are just attached with some pieces of tape so if it's disliked we can easily remove them again. If it's generally liked we could cover them with packing tape to preserve them a bit better.

![Scheppach_wood](https://github.com/comakingspace/do-something/assets/26494083/ed6a0e13-4c78-47e0-a8be-db8f46bba42a)

This one on the front might be distracting, however the only other option is on the left side of the cover which is a bit of an inconvenient place for quick reference.
![Scheppach_metal](https://github.com/comakingspace/do-something/assets/26494083/73164d48-4086-4776-8dde-a1521a56e32e)

Let me know if you generally like it. I can still make one for the other drill press.

**Comment by LukasFreeze** [comment]
Awesome, thank you!

I think it's fine to have something in front, unfortunately the most used
drills (wood) don't fit there but on the other hand wood is also more
forgiving than metal & plastic if you use the wrong speed.

**Comment by kFrolichs** [comment]
Last Saturday I made the speeds chart for the other (less conveniently named) drill press. I found a good spot for the chart on the side of the machine:
![ZJ5116_both](https://github.com/comakingspace/do-something/assets/26494083/1dd3710d-f720-40f7-b9b4-e4b62e88e444)

Furthermore, I've updated this information in the wiki (i.e., the speed charts and where to find them on the machine).
If I'm not mistaking that would mean that the remaining three tasks on this issue would be finished!
